src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -2485,18 +2485,37 @@ public static class ValidationRules
                 (context, annotation) =>
                 {
                     IEnumerable<EdmError> errors;
-                    if (!annotation.Value.TryCast(annotation.Term.Type, out errors))
+                    if (!annotation.UsesDefault)
                     {
-                        foreach (EdmError error in errors)
+                        if (!annotation.Value.TryCast(annotation.Term.Type, out errors))
                         {
-                            if (error.ErrorCode != EdmErrorCode.RecordExpressionMissingRequiredProperty)
+                            foreach (EdmError error in errors)
                             {
-                                context.AddError(error);
+                                if (error.ErrorCode != EdmErrorCode.RecordExpressionMissingRequiredProperty)
+                                {
+                                    context.AddError(error);
+                                }
                             }
                         }
                     }
                 });
 
+        /// <summary>
+        /// Validates that if a vocabulary annotation doesn't have a value, it should declare a type with a default value.
+        /// </summary>
+        public static readonly ValidationRule<IEdmVocabularyAnnotation> VocabularyAnnotationUseDefaultTermMustHaveDefaultValue =
+            new ValidationRule<IEdmVocabularyAnnotation>(

```

Comment:
this rule will have to be updated to use our new interface that contains UsesDefault; otherwise rule looks great.

Reply:
Should I use the new interface in the header (as in ValidationRule<IEdmVocabularyAnnotationWithDefault>) or cast in the body?

Reply:
hmmm... you should be able to use the new interface in the validation rule as long as the interface is declared public.

Reply:
Okay, sounds good, thanks! ✅